frontend/src/pages/Browse/Browse.test.tsx

```diff
@@ -1,30 +1,27 @@
 import { screen } from "@testing-library/react";
+import { vi } from "vitest";
 import { axe } from "vitest-axe";
 
 import { renderWithRouter } from "@/lib/test/renderWithRouter";
 
 import { Browse } from "./Browse";
 
+vi.mock("@/components/ProductColumn", () => ({
+  ProductColumn: () => (
+    <div data-testid="supermarket-container">Mocked ProductColumn</div>
+  ),
+}));
+
```

Comment:
Define variables to use as props 

```suggestion
const nwData = nwFixture;
const pnsData = pnsFixture.results;
const wlData = wlsFixture;
```